src/devices/GamepadMappings.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2019 Immersive Web Community Group. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+Example Gamepad mapping. Any of the values may be omitted for the original
```

Comment:
I'm not as familiar with the gamepad code as the rest of the codebase, but some thoughts:

* `// Chrome 76 Diff: Should be 2` -- what does this mean? Chrome 76 implements it different than spec, so is this patching Chrome's responses to the spec's? What about non-Chrome implementations?
* This is getting pretty large & complex! I wonder due to the robustness of the XRInput having changed over the last year or so, if the gamepad code should be standalone in another directory or something.
* Is there anything to leverage from [THREE.VRController](https://github.com/stewdio/THREE.VRController)? Either something that can be outsourced to that component, or adopted?

Reply:
I ended up removing all the `// Chrome 76` comments and simply making the change in the latest commit in the PR, because this version won't be what the Chrome team is using for the 76 release anyway.